### ISSUE TITLE
Do we need Mediation sdk for Admob reward video?

### ISSUE THREAD

**Issue by yanbaha** [ratson/cordova-plugin-admob-free#322]
Banner ads and interstitial works without problemes.
for reward video ads they work on the emulator of Android studio but nothing is shown on  real device.

my question is: I use only Admob, do I need to install Admob mediation (https://github.com/ratson/cordova-admob-mediation) even for Admob reward videos?

**Comment by diegoestacho999** [comment]
Yes, you do. Without mediations, you will not be able to show your ads. I read this in a documentation of the admob itself.
If you need help, just call.

**Comment by yanbaha** [comment]
Allright, but in the documentation of this Plugin the mediation is only for Facebook and Inmobi ads.
so, what to do for Admob reward video ads?

**Comment by Thomas0921** [comment]
agree, any help will be much appreciated 😢
